autoscheduler/scraper/views.py

```diff
@@ -14,3 +16,16 @@ def get_object(self):
         course_num = self.request.query_params.get('course_num')
         term = self.request.query_params.get('term')
         return Course.objects.get(dept=dept, course_num=course_num, term=term)
+
+class RetrieveSectionView(generics.RetrieveAPIView):
+    """ API endpoint for viewing course information, used by /api/sections.
+        This view returns a serialized section, should return .
+    """
+    serializer_class = SectionSerializer
+
+    def get_object(self):
+        """ Overrides default behavior of get_object() to work without a primary key """
+        dept = self.request.query_params.get('dept')
+        course_num = self.request.query_params.get('course_num')
+        term = self.request.query_params.get('term')
+        return Section.objects.get(dept=dept, course_num=course_num, term=term)
```

Comment:
```suggestion
        return Section.objects.filter(subject=dept, course_num=course_num, term_code=term)
```

Again similar to previous comments, since this is getting multiple objects this should be `Section.objects.filter`. 

Also, the `dept` & `term` field are named `subject`/`term_code` in the `Section` model. No idea why this is the case, but we can change it later.

Reply:
Also, we need a new line at the end of this file.